frontend/src/components/pages/admin/QuestionPage.tsx

```diff
@@ -32,6 +33,8 @@ const QuestionPage = (): React.ReactElement => {
           setShowAddShortAnswerModal,
           showAddMultipleChoiceModal,
           setShowAddMultipleChoiceModal,
+          isMultiSelect,
+          setIsMultiSelect,
```

Comment:
I think we can keep the naming consistent here!

```suggestion
          showAddMultiSelectModal,
          setShowAddMultiSelectModal,
```